service/src/test/resources/ehr/mapper/immunization/expected-output-observation-statement-all-information.xml

```diff
@@ -0,0 +1,31 @@
+<component typeCode="COMP" >
+    <ObservationStatment classCode="OBS" moodCode="EVN">
+        <id root="C93659E1-1107-441C-BE25-C5EF4B7831D1" />
+        <statusCode code="COMPLETE" />
+        <effectiveTime>
+            <center value="20100630055900"/>
+        </effectiveTime>
+        <availabilityTime value="20100630054806" />
+        <pertinentInformation typeCode="PERT">
+            <sequenceNumber value="+1"/>
+            <pertinentAnnotation classCode="OBS" moodCode="EVN">
+                <text>Parent Present: true
```

Comment:
I think these are meant to be output space delimited. Confirm with @dmcavenue1

Reply:
Space delimited, confirmed by @dmcavenue1